client/js/jsonedit.js

```diff
@@ -230,8 +230,8 @@ const jeCommands = [
     forceKey: 'C',
     options: [ { type: 'string', label: 'URL' } ],
     call: async function(options) {
-      const sourceURL = options.URL.replace(/\/[^\/]+$/, a=>`/state${a}`);
-      const targetURL = location.href.replace(/\/[^\/]+$/, a=>`/state${a}`);
+      const sourceURL = options.URL.replace(/\/[^\/]+$/, a=>`${vtt_env.HTTP_URL_PREFIX}/state${a}`);
```

Comment:
This command can download the state from another server so you can't just use the local prefix. The user enters the complete path to the other room though so the original behavior of just adding `/state` should be fine.

Reply:
You are right. Will fix.